### PR TITLE
SEO-2835-ASP.NET-Core-Switch-Interlinking

### PR DESCRIPTION
**Pull request by berylouma**
Hi @ChristoIssac

|Title|Description|
|--|--|
|Task Category|ASP.NET Core Toggle Switch Button Money Keywords|
|Content Task Link/Mail Screenshot|https://syncfusion.atlassian.net/browse/CONTENT-7463|
|UX task|NA|
|Master / Dev /Interlinking PR link|Inter - https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/1115|
|Ticket/Task link|https://syncfusion.atlassian.net/browse/SEO-2835
|Excel/SharePoint link| Retained Keywords, volume, and position. https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/ASP.NET-Toggle-Switch-Button---Money-Keywords.aspx|

| Keyword | Volume | Position | Reason for change | Changes Made |
| ---      | ---       | ---     | ---     | ---     |
|asp.net core toggle switch button |0-10 |1FT|Maintain FT position|Made on-page changes|
|asp.net core toggle button|0-10 |1FT|Maintain FT position|Made on-page changes|
|asp.net switch button|10-100 |3FT |To move FT to the first position|Made on-page changes|
|asp.net toggle button control|0-10 |1FT |To maintain FT position|Made on-page changes|

Worked on 17 keywords in the react dialog and am keeping track of all keyword details in the link below:-https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/Keyword-Plan---ASP.NET-Core-Toggle-Switch-Button-Money-Keywords.aspx
Regards,
Beryl.

[CONTENT-7463]: https://syncfusion.atlassian.net/browse/CONTENT-7463?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SEO-2835]: https://syncfusion.atlassian.net/browse/SEO-2835?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ej2-asp-core-mvc/switch/EJ2_ASP.NETCORE/getting-started.md [hunk 1]
```diff
@@ -149,4 +149,6 @@ This section explains how to set [`onLabel`](https://help.syncfusion.com/cr/aspn
 ## See also
 
 * [Getting Started with Syncfusion ASP.NET Core using Razor Pages](https://ej2.syncfusion.com/aspnetcore/documentation/getting-started/razor-pages/)
-* [Getting Started with Syncfusion ASP.NET Core MVC using Tag Helper](https://ej2.syncfusion.com/aspnetcore/documentation/getting-started/aspnet-core-mvc-taghelper)
\ No newline at end of file
+* [Getting Started with Syncfusion ASP.NET Core MVC using Tag Helper](https://ej2.syncfusion.com/aspnetcore/documentation/getting-started/aspnet-core-mvc-taghelper)
+
+N> You can refer to our [ASP.NET Core Toggle Switch Button](https://www.syncfusion.com/aspnet-core-ui-controls/toggle-switch-button) feature tour page for its groundbreaking feature representations. You can also explore our [ASP.NET Core Toggle Switch Button example](https://ej2.syncfusion.com/aspnetcore/Button/Switch#/material) that shows you how to render the Switch in ASP.NET Core.
```
Comment (Girija-Rajendran) [inline ej2-asp-core-mvc/switch/EJ2_ASP.NETCORE/getting-started.md:154]: Toggle Switch Button in ASP.NET Core
image

Comment (berylouma) [reply ej2-asp-core-mvc/switch/EJ2_ASP.NETCORE/getting-started.md:154]: Changed

**Review by Girija-Rajendran — APPROVED** [review]

File: ej2-asp-core-mvc/switch/EJ2_ASP.NETCORE/getting-started.md [hunk 2]
```diff
@@ -149,4 +149,6 @@ This section explains how to set [`onLabel`](https://help.syncfusion.com/cr/aspn
 ## See also
 
 * [Getting Started with Syncfusion ASP.NET Core using Razor Pages](https://ej2.syncfusion.com/aspnetcore/documentation/getting-started/razor-pages/)
-* [Getting Started with Syncfusion ASP.NET Core MVC using Tag Helper](https://ej2.syncfusion.com/aspnetcore/documentation/getting-started/aspnet-core-mvc-taghelper)
\ No newline at end of file
+* [Getting Started with Syncfusion ASP.NET Core MVC using Tag Helper](https://ej2.syncfusion.com/aspnetcore/documentation/getting-started/aspnet-core-mvc-taghelper)
+
+N> You can refer to our [ASP.NET Core Toggle Switch Button](https://www.syncfusion.com/aspnet-core-ui-controls/toggle-switch-button) feature tour page for its groundbreaking feature representations. You can also explore our [ASP.NET Core Toggle Switch Button example](https://ej2.syncfusion.com/aspnetcore/Button/Switch#/material) that shows you how to render the Toggle Switch Button in ASP.NET Core.
```
Comment (Sharly-Atieno) [inline ej2-asp-core-mvc/switch/EJ2_ASP.NETCORE/getting-started.md:154]: Update the correct demo link her

Comment (berylouma) [reply ej2-asp-core-mvc/switch/EJ2_ASP.NETCORE/getting-started.md]: Done

File: ej2-asp-core-mvc/switch/EJ2_ASP.NETCORE/getting-started.md [hunk 3]
```diff
@@ -149,4 +149,6 @@ This section explains how to set [`onLabel`](https://help.syncfusion.com/cr/aspn
 ## See also
 
 * [Getting Started with Syncfusion ASP.NET Core using Razor Pages](https://ej2.syncfusion.com/aspnetcore/documentation/getting-started/razor-pages/)
-* [Getting Started with Syncfusion ASP.NET Core MVC using Tag Helper](https://ej2.syncfusion.com/aspnetcore/documentation/getting-started/aspnet-core-mvc-taghelper)
\ No newline at end of file
+* [Getting Started with Syncfusion ASP.NET Core MVC using Tag Helper](https://ej2.syncfusion.com/aspnetcore/documentation/getting-started/aspnet-core-mvc-taghelper)
+
+N> You can refer to our [ASP.NET Core Toggle Switch Button](https://www.syncfusion.com/aspnet-core-ui-controls/toggle-switch-button) feature tour page for its groundbreaking feature representations. You can also explore our [ASP.NET Core Toggle Switch Button example](https://ej2.syncfusion.com/aspnetcore/Button/Switch#/bootstrap5) that shows you how to render the Toggle Switch Button in ASP.NET Core.
```
Comment (Sharly-Atieno) [inline ej2-asp-core-mvc/switch/EJ2_ASP.NETCORE/getting-started.md:154]: FT have been interlinked so remove bellow section
You can refer to our [ASP.NET Core Toggle Switch Button](https://www.syncfusion.com/aspnet-core-ui-controls/toggle-switch-button) feature tour page for its groundbreaking feature representations.

**Pull request closed without merging**